docs/content/en/docs/core-concepts/usecase-orchestrate.md

```diff
@@ -98,19 +98,7 @@ file.
 For example, the definition of the `evaluate-dependencies` evaluation
 looks like this:
 
-```yaml
-apiVersion: lifecycle.keptn.sh/v1alpha3
-kind: KeptnEvaluationDefinition
-metadata:
-  name: evaluate-dependencies
-  namespace: simplenode-dev
-spec:
-  objectives:
-    - keptnMetricRef:
-        name: available-cpus
-        namespace: simplenode-dev
-      evaluationTarget: ">4"
-```
+{% include "usecase-orchestrate.md_1.yaml" %}
```

Comment:
does the include actually work? The resource is located in assets folder, so I guess the path should be `assets/usecase-orchestrate.md_1.yaml`. You can try it out locally by executing `make docs-build && make docs-serve`